src/data_provider/src/packages/packageLinuxParserHelper.h

```diff
@@ -133,6 +135,113 @@ namespace PackageLinuxHelper
         return ret;
     }
 
+
+    static nlohmann::json parseSnap(const nlohmann::json& info)
+    {
+        nlohmann::json ret;
+
+        std::string name;
+        std::string version;
+        std::string vendor;
+        std::string install_time;
+        std::string location;
+        std::string architecture;
+        std::string groups;
+        std::string description;
+        std::string priority;
+        std::string multiarch;
+        std::string source;
+        std::string format;
+        int         size = 0;
+
+        bool        hasName    = false;
+        bool        hasVersion = false;
+
+        // TODO: Dismiss record if name or version are not present. 
```

Comment:
I think more than updating the comment, the code must be changed to implement the feature. We can not accept a package without a name or version.

Reply:
> I think more than updating the comment, the code must be changed to implement the feature. We can not accept a package without a name or version.

Please, explain about this, in case of missing name and version values, should it return empty json? e.g.